src/gssapi_ntlmssp.h

```diff
@@ -53,6 +53,17 @@ extern "C" {
 #define GSS_NTLMSSP_RESET_CRYPTO_OID_STRING GSS_NTLMSSP_BASE_OID_STRING "\x03"
 #define GSS_NTLMSSP_RESET_CRYPTO_OID_LENGTH GSS_NTLMSSP_BASE_OID_LENGTH + 1
 
+/* NTLM Get SIDs OID
+ * OID to be used with gss_inquire_sec_context_by_oid()
+ * to get buffer with a list of user's raw SIDs
+ * as reported by authentication process.
+ * MS-PAC used for Kerberos is not fully available with NTLM
+ * so this is the way of getting details for user.
+ */
+
+#define GSS_NTLMSSP_GET_SIDS_OID_STRING GSS_NTLMSSP_BASE_OID_STRING "\x04"
+#define GSS_NTLMSSP_GET_SIDS_OID_LENGTH GSS_NTLMSSP_BASE_OID_LENGTH + 1
```

Comment:
Are we sure all we want to return are sids ?
What about the other fields of the wbcAuthUserInfo struct?

Reply:
List of SIDs is a must for user identity; also user_session_key is already provided separately;
all those strings like full_name and home_directory are nice to have; timestamps are used in very specific cases.

Reply:
Yes I understand that your interest is principally in the SIDs, but I am thinking about the problem in a more general way and want to find a solution that is more generally useful to others as well.

Reply:
Yes, I agree, for general case it's better to export all possible details from wbcAuthUserInfo. But in my code I didn't want to handle serialization/de-serialization to keep it simple enough. If you make implementation with more details, I will be happy to use it :-)